src/features/app/interface.rs

```diff
@@ -1,16 +1,11 @@
 use crate::app::AppContext;
 
-use self::protobuf::{App, GetRatingRequest, GetRatingResponse};
-pub use protobuf::app_server;
+use crate::features::pb::app::{GetRatingRequest, GetRatingResponse};
```

Comment:
This also applies to Line 1 and later lines, but probably best to merge all the `use crate::{...}` into a single line so it's easier to find all imports.